partcad-cli/src/partcad_cli/click/commands/reset.py

```
@@ -5,29 +5,50 @@
-def cli() -> None:
+def cli(repo_only: bool, runtime_only: bool, cache_only: bool) -> None:
     with Process("Reset", "global"):
-        cached_import_types = ["git", "tar"]
-        for import_type in cached_import_types:
-            cache_dir = os.path.join(user_config.internal_state_dir, import_type)
-            if os.path.exists(cache_dir):
-                with Action("Repos", import_type):
-                    shutil.rmtree(cache_dir)
-                    info(f"Removed cached {import_type} dependencies: '{cache_dir}'")
+        if repo_only or not (cache_only or runtime_only):
+            for import_type in cached_import_types:
```

Ruff (0.8.2)

Undefined name `cached_import_types`

(F821)